CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ When updating the changelog, remember to be very clear about what behavior has c
 and what APIs have changed, if applicable.
 
 ## [Unreleased]
+- Observability enhancements for D2 announcer.
```

Comment:
this should go under a heading of a new version instead of the "Unreleased" one. Also, the same version should be updated in the gradle.properties file. You could reference a previous PR, like https://github.com/linkedin/rest.li/pull/710/files.

Detailed steps and instructions can be found in this wiki: https://iwww.corp.linkedin.com/wiki/cf/pages/viewpage.action?spaceKey=ENGS&title=Pegasus+Git+Workflow#PegasusGitWorkflow-Releasing

Reply:
Done